### DISCUSSION TITLE
Files stay hovered on GitHub repositories.

### DISCUSSION THREAD

**Discussion by Lort533** [community/community#52666] [Repositories]
### Select Topic Area

Bug

### Body

Basically, if you go to any GitHub repository and hover over files, the last selected one will remain hovered, even if the mouse is away from the file container. I wanted to try and fix the code manually, but it's TypeScript and I have no idea how to edit it in real time in Chrome.

Bug tested and seen on various browsers (except Safari, because it's too outdated; IE11 is no longer supported and it didn't display correctly for me) using https://www.browserling.com/, and Chrome/Edge on my PC.

https://user-images.githubusercontent.com/76491869/231592363-5447398a-0ee6-44dd-9fbb-ea5cdf1fe8ac.mp4

**Comment by opegamanalitarok** [comment]
To edit TypeScript code in real-time in Chrome, you can use the Chrome Developer Tools. Here's how you can do it:

1. Open the GitHub repository in Chrome.
2. Right-click on the file that you want to edit and select "Inspect" from the context menu.
3. This will open the Chrome Developer Tools. In the "Elements" tab, you should see the HTML and CSS code for the file that you selected.
4. Switch to the "Sources" tab. Here you should see a list of files included in the repository, including any TypeScript files.
5. Locate the TypeScript file that you want to edit and click on it to open it in the editor.
6. You can now edit the code as desired. Changes made in the editor will be reflected in the browser in real-time.

Once you are satisfied with your changes, you can save the edited file to your local machine and then create a pull request to merge the changes into the original repository.
Note that if the changes you make are not reflected in the browser, you may need to refresh the page to see the updated version. Also, keep in mind that making changes directly to the code in this manner is not recommended for production environments and should only be done for testing or debugging purposes.

Hope this helps! 🙌🙌

**Comment by Lort533** [comment]
Hello,
thank you for your reply. I've managed to edit the code (had to enable overwriting folder for it to work), and as I see - currently there is no way of fixing it. There is only `onContainerMouseMove` and `onContainerMouseOver` events, but anything similar to `onContainerMouseAway` event is missing. That would explain why it can't stop being hovered. GitHub development team would have to implement their own optimized way to determine if the mouse if away from container.

TypeScript seems to be only a preview - actual script is obfuscated and changed to JavaScript. Every change has to be made in the obfuscated file, and I can't really be asked to do that, especially if GitHub development team can do that on their side, in an easier way.
